environment/src/main/java/com/sequenceiq/environment/environment/domain/Environment.java

```diff
@@ -347,4 +351,12 @@ public Tunnel getTunnel() {
     public void setTunnel(Tunnel tunnel) {
         this.tunnel = tunnel;
     }
+
+    public Json getParameters() {
```

Comment:
I don't like that we start adding jsons to the relational database when it's unavoidable. Instead add a pojo.

Reply:
Suggestion 2: if you can't let jsons go for some reason, than at least implement it like we did with the `EnvironmentTelemetry` here. Only this class has to know that it has a json representation inside the DB, in the code we want to do json conversions.

Reply:
Suggestion 1 was this: https://stackoverflow.com/a/10434692/3757113

This is better than the json.

Reply:
reworked